### ISSUE TITLE
Review new sections on FAIR and personal story added to RDM chapter

### ISSUE THREAD

**Issue by rainsworth** [alan-turing-institute/the-turing-way#633]
### Summary

Review the content added in https://github.com/alan-turing-institute/the-turing-way/pull/568/ - which was merged before a proper expert review because there were pull requests up the chain that when approved would create lots of merge conflicts.

### What needs to be done?

- [ ] Review the FAIR section and subsections, if it is high-level enough but informative too
- [ ] Review the personal story section

### Who can help?

* @pherterich, @rosiehigman  
* Anyone with FAIR expertise!

---

**Comment by pherterich** [comment]
@rainsworth any idea how to smartly do that now that the PR is merged? Happy to give it a look, but not sure what the best approach is. Maybe just read those sections and create an issue with suggested changes if needed?

**Comment by rainsworth** [comment]
@pherterich yeah, if you just check the diffs on #568 and you can just open your own pull request to make changes.
